### DISCUSSION TITLE
[1장] 스터디 진행 회고

### DISCUSSION THREAD

**Discussion by punchdrunkard** [prgrms-web-devcourse/We-are-the-MASTERS-OF-HTTP#40] [essay]
# 1장 내용적인 측면
- 챕터 4(커넥션 관리)를 제외하고는 무난한 내용이였다.
  - 4장이 어렵게 느껴진 이유
    - 프론트엔드 측에서 사용하지 않는 부분이라 와닿지 않았다.
    - 전공적인 지식이 결합된 부분이라 전공 공부한 느낌이 있었다. 깊이를 적당히 조절하며 읽을 필요가 있을 것 같다.
- 반면 2,3 장의 경우 실제 프론트엔드 개발에서 많이 사용하는 내용이라서 기억에 남았고 유용하였다.

# 스터디 진행 측면
## 좋았던 점
- 승희님이 외부 자료를 많이 가져와서, 책 내용을 넘어서 실제 개발에 사용되는 내용에 대한 이야기를 할 수 있어서 좋았다.
  - 따라서, 퀴즈를 낼 때 책 내용 뿐만이 아니라 책 내용에 대해 본인이 찾아본 다른 내용에 대한 퀴즈를 내는 방식으로 범위를 넓히면 좋을 것 같다.
- 해경님이 책 내용에 대한 CS 기반 지식에 대한 문제를 내주셔서, CS 기초에 대해서 다시 복습할 수 있는 기회가 되어서 좋았다.

## 아쉬웠던 점
- 생각보다 한 장, 한 장 볼륨이 커서 부담이 있었다. (특히 이번 주 6,7 장이 가장 힘들었다) 
- 책 내용에 따라 분량을 조절할 필요성이 있다.

**Comment by Hyevvy** [comment]
화이팅 (ง •̀ㅁ•́)ง✧
